Train Begin...
Episodes:   0%|                                                                   | 0/1000 [00:06<?, ?it/s]
Traceback (most recent call last):                                                                         
경로 재계산 실패: AGV F, 현재 위치 (49, 12), 시작 위치 (40, 3), 목표 위치 (40, 3)
경로 재계산 실패: AGV O, 현재 위치 (69, 12), 시작 위치 (60, 3), 목표 위치 (60, 3)
경로 재계산 실패: AGV F, 현재 위치 (49, 12), 시작 위치 (40, 3), 목표 위치 (40, 3)
경로 재계산 실패: AGV I, 현재 위치 (65, 69), 시작 위치 (53, 89), 목표 위치 (53, 89)
경로 재계산 실패: AGV O, 현재 위치 (69, 12), 시작 위치 (60, 3), 목표 위치 (60, 3)
경로 재계산 실패: AGV F, 현재 위치 (49, 12), 시작 위치 (40, 3), 목표 위치 (40, 3)
경로 재계산 실패: AGV I, 현재 위치 (65, 69), 시작 위치 (53, 89), 목표 위치 (53, 89)
경로 재계산 실패: AGV N, 현재 위치 (85, 18), 시작 위치 (90, 3), 목표 위치 (90, 3)
경로 재계산 실패: AGV O, 현재 위치 (69, 12), 시작 위치 (60, 3), 목표 위치 (60, 3)
  File "/home/js/DAA_CPS/train.py", line 109, in <module>
    train(config)
  File "/home/js/DAA_CPS/train.py", line 59, in train
    action = agent.act(state, env, agv_id)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/js/DAA_CPS/Agent.py", line 80, in act
    q_values = self.policy_net(state)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/js/anaconda3/envs/DAA_CPS/lib/python3.12/site-packages/torch/nn/modules/module.py", line 1739, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/js/anaconda3/envs/DAA_CPS/lib/python3.12/site-packages/torch/nn/modules/module.py", line 1750, in _call_impl
    return forward_call(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/js/DAA_CPS/Agent.py", line 26, in forward
    x = F.relu(self.fc1(x))
        ^^^^^^^^^^^^^^^^^^^
  File "/home/js/anaconda3/envs/DAA_CPS/lib/python3.12/site-packages/torch/nn/functional.py", line 1704, in relu
    result = torch.relu(input)
             ^^^^^^^^^^^^^^^^^
KeyboardInterrupt
